index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en"> 
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta property="og:image" content="./desktop_screenshot.png">
+  <title>Golden View School</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/f8b4998ade.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="pre-nav">
+    <a href=""> <i class="fab fa-twitter"></i></a>
+    <a href=""><i class="fab fa-instagram"></i></a>
+    <ul class="pre-menu">
+      <li><a href="#">English</a></li>
+      <li><a href="#">My Page</a></li>
+      <li><a href="#">Logout</a></li>
+    </ul>
+  </div>
+  <nav id="nav" class="invisible">
```

Comment:
- [ ] I kindly suggest that you ensure that the background of your navbar is transparent or the same color/pattern as the top section background

<img width="291" alt="image" src="https://user-images.githubusercontent.com/52934128/196702039-1f28ad3e-b5b4-4acc-bfb0-23ceed375791.png">